exporters/trace/jaeger/uploader.go

```diff
@@ -114,30 +113,30 @@ func WithAttemptReconnectingInterval(interval time.Duration) AgentEndpointOption
 	}
 }
 
-// WithCollectorEndpoint defines the full url to the Jaeger HTTP Thrift collector.
-// For example, http://localhost:14268/api/traces
-func WithCollectorEndpoint(collectorEndpoint string, options ...CollectorEndpointOption) EndpointOption {
+// WithCollectorEndpoint defines the full url to the Jaeger HTTP Thrift collector. This will
+// use the following environment variables for configuration if no explicit option is provided:
+//
+// - OTEL_EXPORTER_JAEGER_ENDPOINT is the HTTP endpoint for sending spans directly to a collector
+// - OTEL_EXPORTER_JAEGER_USER is the username to be sent as "Basic" authentication to the collector endpoint.
```

Comment:
```suggestion
// - OTEL_EXPORTER_JAEGER_USER is the username to be sent as authentication to the collector endpoint.
```

> Remove comments outlining what the userid, password needs to be for Jaeger (basic, basic).

Is this the change you are referencing, not sure what comments you suggest removing?